suite/tests/CMakeLists.txt

```diff
@@ -4044,6 +4036,19 @@ if (BUILD_CLIENTS)
         "${prefetch_analyzer_path}@-trace_dir@${testname_full}.*.dir/trace")
     endif ()
 
+    # Stress-test drcachesim with drstatecmp checks.
+    if (AARCH64)
+      torunonly_drcachesim(drstatecmp-fuzz drstatecmp-fuzz-app
+        "-trace_after_instrs 10M -enable_drstatecmp" "" "")
+    endif ()
+    if (NOT WIN32)
+      torunonly_drcachesim(drstatecmp-delay-simple ${ci_shared_app}
```

Comment:
Would be better to rename this old one. Otherwise it looks like both of these tests are supposed to be exactly the same except online vs offline. Maybe drstatecmp-delay-early-exit?

Reply:
I'm actually not sure why this has the exit flag: without the flag it also hits the bug so I wonder if someone added it to mask the bug??  Or to reduce test runtime and never noticed the bug.  Maybe I should just remove the flag.

Reply:
I removed it.  If the tests are green we'll go with that.